assets/js/atomic/utils/create-blocks-from-template.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Creates blocks for a given inner blocks Template.
+ *
+ * @param {Array} template Inner Blocks Template.
+ */
+export const createBlocksFromTemplate = ( template ) => {
```

Comment:
This utility will create blocks recursively from a defined default template. Used currently to reset the layout back to default.